src/shared/highlighted-field/highlighted-field-context.js

```diff
@@ -1,12 +1,12 @@
 import { createContext } from 'react'
 
-export const CurrentItemContext = createContext({
+export const HighlightedFieldIdsContext = createContext({
```

Comment:
Why is `ids` plural? From the name it seems like this contain multiple `ids`.

Reply:
Because a highlighted field's "id" is essentially the combination of the data element's id as well as the coc's id

Reply:
I see where you're coming from, but we're referring to that concatenation as the `fieldId`. Check `get-field-id.js`, and functon `getFieldId()`. So I would still argue for this to be called `fieldId`.

Reply:
Ah, fair enough, will change it